content/3.Joining Corda Network/4.Onboarding Workflow.md

```diff
@@ -33,29 +39,35 @@ i. The node operator fills out fields in bold in the table below, for every node
 | **Organisation Unit (OU)**  |     - |
 | State (ST)                  |     - |
 | Common Name (CN)            |     - |
-| Node Operator email address |     - |
-
 
-ii. CNF performs legal identity and sanctions checks on Organisation field (O) for each node.
-
-iii. CNF needs to verify the person submitting this CSR has access to the associated email inbox and will do so by sending an email to the ‘node operator email address’ and waiting for a confirmation reply.
+- "O" Organisation should be the same Legal Entity as what you have filled out in your Terms of Use (Step 1), unless you are onboarding for your sponsee. This is also the owner of the assets being used on Corda Network. For guidance on which legal entity name to pick for your node, see the [Allowable Identity Names](https://corda.network/corda-network-rulebook/allowable-identity-names-on-corda-network) session.
```

Comment:
it seems like autocorrect is changing "section" into "session"

Reply:
Updated